test/plugin/scenarios/solon-2.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,103 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+segmentItems:
+- serviceName: solon-2.x-scenario
+  segmentSize: ge 2
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: H2/JDBC/PreparedStatement/executeQuery
+      parentSpanId: 0
+      spanId: 1
+      spanLayer: Database
+      startTime: not null
+      endTime: not null
+      componentId: 32
+      isError: false
+      spanType: Exit
+      peer: not null
+      tags:
+      - {key: db.type, value: not null}
+      - {key: db.instance, value: test}
+      - {key: db.statement, value: not null}
+      skipAnalysis: 'false'
+    - operationName: '/testcase/error'
+      parentSpanId: 0
+      spanId: 2
+      spanLayer: Http
+      startTime: not null
+      endTime: not null
+      isError: false
+      componentId: not null
+      tags:
+      - {key: url, value: not null}
+      - {key: http.method, value: GET}
+      skipAnalysis: 'false'
+    - operationName: 'Solon:GET:/testcase/error'
```

Comment:
None of entry spans have reference verifications, this test scenarios can't prove the tracing continues.

Solon is a RPC framework AFAIK, your test doesn't cover an important part.

Reply:
Sorry, I can't fully understand the meaning of automated test scripts. I wrote this script with reference to spring. From the results I get from running it, I think I can get the link tracing information

Reply:
If no reference, I can't see how it links. 

You should check the segments of traces, and understand the data.

As I have explained, we don't maintain the plugins, but host for you(for example). After this, I can't ask you every time when someone submitted an issue/discussion. So, the best way is, test scenarios themselves can prove it works as expected.